### DISCUSSION TITLE
HAProxy and admin log

### DISCUSSION THREAD

**Discussion by netbja** [roxy-wi/roxy-wi#350] [Ideas]
It's possible to have admin log with HAPRoxy
Ex: 
`local0.notice /var/log/haproxy/haproxy-admin.log
`
Are you able to add this feature ?
https://www.haproxy.com/documentation/hapee/latest/observability/logging/overview/

**Comment by Aidaho12** [comment]
Yes, I guess it possible. Will research it

**Comment by Aidaho12** [comment]
Hello, it will be done soon https://roxy-wi.org/changelog#6_3_6

**Reply by netbja** [reply]
Thank you for this very responsive support !

**Reply by Aidaho12** [reply]
The changes are here https://github.com/hap-wi/roxy-wi/commit/810a9d9460ea44a9aac6f93c65d76f9a440dc6e1
